HTTP Error 404: Not FoundTraceback (most recent call last):
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\WaveDownload/python/retrieve-waves.py", line 84, in <module>
    main()
  File "C:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\Modules\WaveDownload/python/retrieve-waves.py", line 79, in main
    bomFile = wget.download(url, out=os.path.join(workDir,'ncFiles'))
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\site-packages\wget.py", line 526, in download
    (tmpfile, headers) = ulib.urlretrieve(binurl, tmpfile, callback)
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\urllib\request.py", line 248, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\urllib\request.py", line 223, in urlopen
    return opener.open(url, data, timeout)
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\urllib\request.py", line 532, in open
    response = meth(req, response)
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\urllib\request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\urllib\request.py", line 570, in error
    return self._call_chain(*args)
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\urllib\request.py", line 504, in _call_chain
    result = func(*args)
  File "c:\Users\z3531278\Documents\01_FEWS-RegionHome-Aus\bin\windows\python\bin\conda-venv\lib\urllib\request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found
